pkg/test/main_entry.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"flag"
+	"log"
+	"os"
+	"testing"
+)
+
+func MainEntry(m *testing.M) {
+	projRoot := flag.String("root", "", "path to project root")
```

Comment:
We should probably declare the flag names as constants here and then import them in `commands/operator-sdk/cmd/test.go` so we don't have to change them in two places.